src/sql/src/plan/transform_ast.rs

```diff
@@ -392,6 +406,53 @@ impl<'ast> VisitMut<'ast, Aug> for FuncRewriter<'_> {
         }
     }
 
+    fn visit_table_with_joins_mut(&mut self, item: &'ast mut TableWithJoins<Aug>) {
+        visit_mut::visit_table_with_joins_mut(self, item);
+        match &mut item.relation {
+            TableFactor::Function {
+                function,
+                alias,
+                with_ordinality,
+            } => {
+                self.rewriting_table_factor = true;
+                // Functions that get rewritten must be rewritten as exprs
+                // because their catalog functions cannot be planned.
+                if let Some((ident, expr)) = self.rewrite_function(function) {
+                    let mut select = Select::default().project(SelectItem::Expr {
+                        expr,
+                        alias: Some(match &alias {
+                            Some(TableAlias { name, columns, .. }) => {
+                                columns.get(0).unwrap_or(name).clone()
```

Comment:
These lines were not covered at all by any test case (neither unit nor end-to-end test) at commit `2f5d198`. Would it make sense to add a test for these? (Please ignore if no longer uncovered with latest test changes.)